packages/eslint-config-hmpps/index.js

```diff
@@ -0,0 +1,147 @@
+const _import = require("eslint-plugin-import")
+const noOnlyTests = require("eslint-plugin-no-only-tests")
+
+const { fixupPluginRules } = require("@eslint/compat")
+
+const globals = require("globals")
+const typescriptEslint = require("@typescript-eslint/eslint-plugin")
+const tsParser = require("@typescript-eslint/parser")
+const js = require("@eslint/js")
+
+const { FlatCompat } = require("@eslint/eslintrc")
+
+const { rules: bestPracticesRules } = require('./rules/best-practices')
+const { rules: errorsRules } = require('./rules/errors')
+const { rules: nodeRules } = require('./rules/node')
+const { rules: styleRules } = require('./rules/style')
+const { rules: variablesRules } = require('./rules/variables')
+const { rules: es6Rules } = require('./rules/es6')
+const { rules: importsRules } = require('./rules/imports')
+const { rules: strictRules } = require('./rules/strict')
+
+const airBnbRules = Object.assign({}, bestPracticesRules, errorsRules, nodeRules, styleRules, variablesRules, es6Rules, importsRules, strictRules)
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+})
+
+module.exports = [
+  {
+    ignores: [
+      "**/node_modules",
+      "**/public",
+      "**/assets",
```

Comment:
i'm unsure about the `**` in these paths. for "node_modules", it's of course fine, but i would think it's better to specifically point to other folders to ignore. it's perfectly possible for someone to make a path with "assets" in their actual application code.

also, /public does not exist in the template repo nor does the build process generate it

Reply:
oh and another also ;) why ignore the assets folder? if we have enforced styles, surely we want to enforce them across the board?

Reply:
I'm gonna merge these initial changes in then get onto these and you're correct it means our JS doesn't get linted for one thing!